bin/mail/2016-03-21-teacher-recruitment/generate-k5-workshop-attendees

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+attendees = query_subscribed_contacts(q: "kind_s:ProfessionalDevelopmentWorkshopSignup")
+
+export_contacts_to_csv attendees, "k5-workshop-attendees.csv"
```

Comment:
It is supposed to be "Teachers who attended K-5 workshops", but we don't actually track attendance for the K5 workshops. I just clarified with PLC and they would prefer using the teachers who joined the section rather than just signed up. It's still not exactly attendance, but closer. 

I'll update this query.